clang/include/clang/DPCT/DPCTOptions.inc

```diff
@@ -416,4 +416,11 @@ DPCT_NON_ENUM_OPTION(DPCT_OPT_TYPE(static llvm::cl::opt<bool>), GenHelperFunctio
         llvm::cl::desc("Generate helper function files in the --out-root directory. Default: off."),
         llvm::cl::cat(DPCTCat), llvm::cl::init(false))
 
+DPCT_NON_ENUM_OPTION(DPCT_OPT_TYPE(static llvm::cl::opt<std::string>),
+                     QueryAPIMapping, "query-api-mapping",
+                     llvm::cl::desc("Outputs to stdout functionally compatible "
+                                    "SYCL API mapping for CUDA API."),
```

Comment:
Query functionally compatible SYCL API to migrate CUDA API

Reply:
fixed